timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -1432,6 +1432,7 @@ def select_target(ts_df: TimeSeriesDataFrame) -> TimeSeriesDataFrame:
         base_models = trainer.get_model_names(level=0)
         pred_proba_dict_val: Dict[str, List[TimeSeriesDataFrame]] = {
             model: trainer._get_model_oof_predictions(model) for model in base_models
+            if "_FULL" not in model
         }
```

Comment:
Maybe let's also rename the variables `base_models` -> `base_model_names` and `model` -> `model_name` to make it clearer what the variables mean?